evap/results/templates/distribution_bar.html

```diff
@@ -1,11 +1,38 @@
 {% load results_templatetags %}
+{% load evaluation_filters %}
 
 {% spaceless %}
-<div class="distribution-bar">
-    {% for answer, value in distribution.items %}
-        {% if value != 0 %}
-            <div class="vote-bg-{{ answer }}{% if participants_warning %} participants-warning{% endif %}" style="width: {% widthratio value distribution.values|sum 100 %}%;">&nbsp;</div>
+{% if average %}
```

Comment:
this cannot happen if included from result_bar.html, right?

Reply:
result bar sets `average=result.average` when including. so `average` exists whenever there is data to display - otherwise the grey disabled bars are shown.